src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -15,12 +15,20 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ProjectGuid>{9E50E7BF-CD6E-4269-A6DD-59FD0BD6C0FD}</ProjectGuid>
     <CommonPath Condition="'$(CommonPath)' == ''">..\..\Common\src</CommonPath>
-    <IsNETNativeLibrary Condition="'$(TargetGroup)' == 'netcore50'">true</IsNETNativeLibrary>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(IsNETNativeLibrary)' == 'true'">
+
+  <!-- Default configurations to help VS understand the options -->
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50_Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'dnxcore50_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'dnxcore50_Release|AnyCPU'" />
+
+  <PropertyGroup Condition="'$(TargetGroup)' == 'netcore50'">
     <DefineConstants>$(DefineConstants);FEATURE_NETNATIVE</DefineConstants>
```

Comment:
In a future PR (not this one), I would like to change the constant FEATURE_NETNATIVE to be 'netcore50' and FEATURE_CORECLR to be dnxcore50.  Part of the rename goal was to try to use consistent naming across MSBuild properties, conditional compile if's, files, etc.  Just something to think about for the next sets of PR's.